src/core/demangle.d

```diff
@@ -2560,6 +2566,7 @@ version(unittest)
     foreach( i; staticIota!(table.length) )
     {
         enum r = demangle( table[i][0] );
+        // NOTE: can't be run at compile time anymore
```

Comment:
That would probably be a nogo, but would also be avoided by a better separation.

Reply:
actually this comment was a leftover, i had fixed that issue in the meantime; removed comment.